crp_APL/crp_msgs/msg/TrafficRule.msg

```diff
@@ -3,4 +3,6 @@ std_msgs/Header header
 # TODO: Define constants for lane edge type (e.g. uint8 type_name=0)
 
 geometry_msgs/PoseWithCovariance stop_pose
-uint8 lane_edge_type
\ No newline at end of file
+uint8 lane_edge_type_left
+uint8 lane_edge_type_right
+uint8 maximum_speed
```

Comment:
float32?

Reply:
I'll change it. I think a speed constraint either from traffic rules or manual input would be an integer number but it's better to be prepared.